docs/content/specs/shared/_index.md

```diff
@@ -111,7 +111,8 @@ These will also be provided as a comment on the module proposal, once accepted,
 
 {{< /hint >}}
 
-Modules **MUST** provide the capability to collect deployment/usage telemetry, via a blank ARM deployment, as detailed in [Telemetry](/Azure-Verified-Modules/help-support/telemetry/) further.
+Modules **MUST** provide the capability to collect deployment/usage telemetry, via a blank ARM deployment, as detailed in [Telemetry](/Azure-Verified-Modules/help-support/telemetry/) further. Telemetry data should be collected on the "top level" `main.bicep` or `main.telemetry.tf` file; it is not necessary to include it in any nested files (child modules).
```

Comment:
@matt-FFFFFF, @mbilalamjad, @segraef - this extra sentence was added for clarity (see the parallel change for the Bicep documentation in the same PR), as there was confusion about whether the telemetryId should be added to the child modules too (if any). Please review/comment/approve as needed. Thanks!

Reply:
lgtm, waiting for @matt-FFFFFF

Reply:
Also LGTM